src/Tribe/Main.php

```diff
@@ -1594,9 +1594,7 @@ public function saved_organizers_dropdown( $current = null, $name = 'organizer[O
 		 */
 		public function admin_body_class( $classes ) {
 			global $current_screen;
-			if ( isset( $current_screen->post_type ) &&
-				 ( $current_screen->post_type == self::POSTTYPE || $current_screen->id == 'settings_page_tribe-settings' )
-			) {
+			if ( isset( $current_screen->post_type ) && ( $current_screen->post_type == self::POSTTYPE || $current_screen->id == 'settings_page_tribe-settings' ) ) {
```

Comment:
Seems like a debate over the best wrong way to do something. Why not break the conditional out into a separate function? `if ( $this->is_events_admin_screen() )`